src/commcare_cloud/ansible/deploy_postgres.yml

```diff
@@ -22,6 +22,7 @@
   roles:
     - {role: pgbouncer, tags: 'pgbouncer'}
 
+# Notes this should be filtered out if the environment does not have remote postgres
```

Comment:
right now this gets deployed to all servers even if we don't have anything in remote_postgresql @dannyroberts was that intentional?

Reply:
No that's not intentional, good catch. In it's original version, this only had one task, which ran with an empty `with_items` if there was nothing in `remote_postgresql`. But then on May 20 I added the "Install python-psycopg2 to be able to set up remote postgresql" task to fix the install (something must have previously installed this on django_manage that doesn't anymore), not realizing that that adds an unnecessary install when there's nothing in `remote_postgresql`. If adding `when: groups.remote_postgresql is defined` or the the like to the block works, I support that